integrated_channels/canvas/client.py

```diff
@@ -93,6 +94,16 @@ def course_update_endpoint(canvas_base_url, course_id):
             course_id,
         )
 
+    @staticmethod
```

Comment:
Since you waited for Binod- he moved the static methods to a utils class.

Reply:
I'll move the other static methods as well then.